docs/07-advanced-usage.md

```diff
@@ -178,3 +178,17 @@ No browser today supports importing an image directly from JS. Instead, you'll w
 
 1. **Recommended!** Keep referencing images by URL. You can put any image URL directly into an `<img>` src tag: `<img src="/img/photo.png">`.
 
+### Custom Rollup Plugins
+
+As long as you’re dealing with valid ES Modules, Snowpack should never need any help. However, there may be times where you need Snowpack to handle non-spec files for specific ecosystems (such as `*.svelte` files). In these edge cases, you can specify custom Rollup plugins in your [Snowpack config](#configuration-options):
```

Comment:
Also trying to communicate “no srsly don’t add this unless you need to” here

Reply:
Haha +1! I took an editing pass at this to make it a bit more concise, but feel free to edit my edits :)

> You may encounter non-JS npm packages that can only run with additional parsing/processing. Svelte packages, for example, commonly include ".svelte" files that will require additional tooling to parse and install for the browser.

> In these rare cases, you can add your own Rollup plugins to your [Snowpack config](#configuration-options) to handle these special files:

Reply:
Sounds good to me!

Reply:
LGTM